lib/os_io.c

```diff
@@ -769,6 +892,7 @@ ___SCMOBJ ___device_select
          nb_read_devs,
          nb_write_devs,
          timeout)
+___processor_state *___ps;
```

Comment:
that looks like a leftover from my initial attempts to pass processor state around; will remove.